.github/workflows/update-engines-version.yml

```diff
@@ -0,0 +1,73 @@
+name: Update Engines Version
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "Version to check and update the engines version"
+        required: true
+
+jobs:
+  update_engines:
+    name: "Check and update @prisma/engines@${{ github.event.inputs.version }}"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Use Node.js
+        uses: actions/setup-node@v2
+
+      - uses: pnpm/action-setup@v2.0.1
+        with:
+          version: 6.9.1 
+          
+      # This step uses `@prisma/ensure-npm-release` (abbv. `enr`) https://github.com/prisma/ensure-npm-release
+      - name: Check if version is available on npm
+        run: |
+          echo 'Checking @prisma/engines@${{ github.event.inputs.version }}'
+          pnpx -y -p @prisma/ensure-npm-release enr update -p @prisma/engines -u ${{ github.event.inputs.version }}
+          pnpx -y -p @prisma/ensure-npm-release enr update -p @prisma/engines-version -u ${{ github.event.inputs.version }}
```

Comment:
These commands are blocking until they succeed?

Reply:
Yes

Reply:
How long do they wait? Endless / until the Action times out?

Reply:
I use exponential backoff so maximum is 30 minutes. If actions times out before that then its another case.